stories/components/Menu/MenuStyled.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+type MenuProps = {
+  hasText?: boolean;
+};
+export const MenuStyled = styled.div<MenuProps>`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-shadow: 2px 0px 3px rgba(0, 0, 0, 0.25);
+  ${(props) =>
+    props.hasText
+      ? `width: 200px;
+        height: 960px;
+        `
+      : `width: 80px;
+        height: 960px;
+        `};
+`;
```

Comment:
```suggestion
  width: ${(props) => props.hasText ? '200px' : '80px')};
  height: 960px
```
Viết như này nhìn đẹp hơn